### DISCUSSION TITLE
[使用方法]：使用pnpm安装的Wot Design Uni， 在使用反馈组件的时候，下方代码不能找到资源 ：import { useToast } from '@/uni_modules/wot-design-uni'，因为uni_modules中没有文件，这种怎样使用呢

### DISCUSSION THREAD

**Discussion by BeforeWinterComes** [Moonofweisheng/wot-design-uni#531] [General]
[使用方法]：使用pnpm安装的Wot Design Uni， 在使用反馈组件的时候，下方代码不能找到资源 ：import { useToast } from '@/uni_modules/wot-design-uni'，因为uni_modules中没有文件，这种怎样使用呢

**Comment by Moonofweisheng** [comment]
参考[常见问题](https://wot-design-uni.pages.dev/guide/common-problems.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E7%BB%84%E4%BB%B6%E5%BA%93%E6%96%87%E6%A1%A3%E4%B8%AD%E9%83%BD%E6%98%AF%E4%BB%8E-uni-modules-wot-design-uni%E5%AF%BC%E5%85%A5%E6%96%B9%E6%B3%95%E5%92%8C%E5%B7%A5%E5%85%B7%E7%B1%BB)

**Comment by BeforeWinterComes** [comment]
好的 谢谢
